metagraph/docs/source/quick_start.rst

```diff
@@ -166,6 +174,18 @@ hence should be dropped to keep the k-mer index free of the non-existent k-mers.
 
     zless SRR403017_clean_contigs.fasta.gz
 
+.. note:: The default parameters in ``metagraph clean`` correspond to no cleaning. That is, an equivalent of ``metagraph transform --to-fasta``, which extracts from the input de Bruijn graph contigs including all its k-mers.
+
```

Comment:
"which extracts from the input de Bruijn graph contigs including all its k-mers." --> "which extracts from the input de Bruijn graph all contigs, without removing any k-mers."